tests/unit/test_utils.py

```diff
@@ -850,3 +850,49 @@ def test_import_member_no_such_class():
             tmt.utils.GeneralError,
             match=r"No such member 'NopeDoesNotExist' in module 'tmt\.steps\.discover'."):
         tmt.plugins.import_member('tmt.steps.discover', 'NopeDoesNotExist')
+
+
+def test_common_base_inheritance(root_logger):
+    """ Make sure multiple inheritance of ``Common`` works across all branches """
+
+    class Mixin(_CommonBase):
+        def __init__(self, **kwargs):
+            super().__init__(**kwargs)
+
```

Comment:
Does it make sense to also `assert type(self).__mro__ == (...)` here? Obviously it would be different for each instantiation of `ClassA` and `ClassB`, so it would have to be parametrized for each run...

Reply:
Isn't testing the whole `__mro__` of `ClassA` and `ClassB` enough?

Reply:
Yeah, probably.